src/main/java/frc/config/Config.java

```diff
@@ -34,6 +50,11 @@ public static String getString(String key) {
 		return result;
 	}
 
+	/**
+	 * Get the double config value corresponding to the key passed in.
+	 * 
+	 * @return The corresponding double value, or 0.0 if the key was invalid
```

Comment:
0.0 seems like a really important valid value. @audrey-xie, please consider changing the invalid key indicator.